Firestore/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Firestore API Branch
```

Comment:
Please merge firestore-api-changes and you'll get a new Unreleased block; all the stuff going out in the forthcoming 0.9.4 has been separated.